packages/@lwc/engine/src/3rdparty/snabbdom/types.ts

```diff
@@ -54,6 +54,11 @@ export interface VCustomElement extends VElement {
     mode: 'closed' | 'open';
     ctor: any;
     clonedElement?: undefined;
+    // This VCustomElement may be reused when is part of the slot content in another slotted VCustomElement.
```

Comment:
@jodarove this is the text that will show up when you roll over the key in VSCode, make it smaller, something like, "copy of the last list of children collection"